packages/itwinui-react/src/core/ThemeProvider/ThemeProvider.tsx

```diff
@@ -164,13 +180,17 @@ export const ThemeProvider = React.forwardRef((props, forwardedRef) => {
   themeOptions.highContrast ??=
     themeProp === 'inherit' ? parent.highContrast : undefined;
 
+  const future = futureProp ?? {};
+  future.themeBridge ??=
```

Comment:
Just out of curiosity, why do we need a `??=` operator here?

Reply:
A related question: why are we using `futureProp ?? {}` here instead of setting the default value to `{}`?

Reply:
I'm asking about the default value, similar to how we do for `themeOptions` here:

https://github.com/iTwin/iTwinUI/blob/be0762e9c938a5b522b06cdb25df8846da9beaa5/packages/itwinui-react/src/core/ThemeProvider/ThemeProvider.tsx#L161

We could set `future = {}` when destructuring from `props`, which would allow using `future.themeProps ??=` (because `future` will always be defined).